public/electron.js

```diff
@@ -217,11 +225,7 @@ ipc.on('data', (event, args) => {
 // Save Video
 ipc.on('save_video', (event, fileName, buffer) => {
   if (VIDEO){
-    const desktop = app.getPath('desktop')
```

Comment:
this is a nit I didn't really care about on the task, but for honeycomb I think we should fix.  We have a global variable `fileName`, but this handler shadows it.  Can we change the variable in the `save_video` function to be `videoFileName` or similar?